appcues/src/main/java/com/appcues/trait/ContentWrappingTrait.kt

```diff
@@ -22,6 +22,11 @@ internal interface ContentWrappingTrait : ExperienceTrait {
      */
     @Composable
     fun WrapContent(
-        content: @Composable (modifier: Modifier, containerPadding: PaddingValues, safeAreaInsets: PaddingValues) -> Unit
+        content: @Composable (
+            modifier: Modifier,
+            containerPadding: PaddingValues,
+            safeAreaInsets: PaddingValues,
+            hasVerticalScroll: Boolean,
```

Comment:
The only change here is the new `hasVerticalScroll` flag, which is passed in from the wrapping trait (modal, tooltip, embed) and propagated all the way through into the StepComposition now. That is the spot where `rememberScrollState()` is now called, giving each step its proper scroll state.